src/MovieCard/MovieCard.css

```diff
@@ -4,15 +4,18 @@
   flex-direction: column;
   text-align: center;
   width: 20vw;
+  min-width: 244px;
```

Comment:
I love the min-width, min-height for making pages responsive! It is simple and straightforward!

Reply:
I like it a lot too!